gremlin-python/src/main/python/gremlin_python/driver/client.py

```diff
@@ -180,25 +181,21 @@ def submit_async(self, message, bindings=None, request_options=None):
             raise Exception("Client is closed")
 
         log.debug("message '%s'", str(message))
-        args = {'gremlin': message, 'aliases': {'g': self._traversal_source}}
-        processor = ''
-        op = 'eval'
+        fields = {'g': self._traversal_source}
         if isinstance(message, traversal.Bytecode):
-            op = 'bytecode'
-            processor = 'traversal'
+            fields['gremlinType'] = 'bytecode'
```

Comment:
Shouldn't need to set gremlinType as server should be able to determine this from the type of the serialized query.

Reply:
It doesn't look like server reads it at this point, so I'll keep these in for now. It will need to be updated once we remove bytecode from python anyways.